### ISSUE TITLE
Move ⭐️ starred listed blog to top of list

### ISSUE THREAD

**Issue by OddHouseGames** [standardnotes/listed#160]
As listed.io is the main landing page for the micro blogging service, it would be beneficial to move the ⭐️ starred Listed blog to the top of the list for easy access to new guests, visitors and returning members, instead of having to search for it each time.

**Comment by atmoio** [comment]
Yeah I was thinking about something like this as well. Perhaps featured blogs can be randomized to appear somewhere in at least the first 10 cards.
